### ISSUE TITLE
Mac failures have exploded

### ISSUE THREAD

**Issue by philberty** [Rust-GCC/gccrs#1675]
We now have:

```
# of expected passes		5320
# of unexpected failures	1940
# of unexpected successes	7
# of expected failures		45
# of unresolved testcases	399
```

Since 0a59abbfc74c3961fb01323d9fbff8427518fde6 it was:

```
# of expected passes		6913
# of unexpected failures	35
# of expected failures		52
```

**Comment by philberty** [comment]
This is likey because of:

https://github.com/Rust-GCC/gccrs/pull/1635

**Comment by philberty** [comment]
fixed in #1676
